File: yes.rs
use std::env;

fn main() {
    let args: Vec<String> = env::args().collect();
    let args = &args[1..];

    let output = if args.is_empty() { "y".to_string() } else { args.join(" ") };
    loop {
        println!("{}", output)
    }
}

File: yes.dart
void main(List<String> arguments) {
  var output = "";
  
  if (arguments.length > 0) {
    for (var args in arguments)
      output += args + ' ';
  }
  else
    output = 'y';
  
  while (true) {
    print('$output');
  }
}
